Quake/Makefile

```diff
@@ -70,7 +70,6 @@ define do_strip
 	$(call cmd_strip,$(1));
 endef
 endif
-CFLAGS += -std=gnu99
```

Comment:
[I haven't seen C89 added to a project in a long while](https://i.imgflip.com/wppxk.jpg)